ci/release/update-version.sh

```diff
@@ -11,7 +11,7 @@
 NEXT_FULL_TAG=$1
 
 # Get current version
-CURRENT_TAG=$(git tag | grep -xE 'v[0-9\.]+' | sort --version-sort | tail -n 1 | tr -d 'v')
+CURRENT_TAG=$(cat VERSION)
```

Comment:
This change actually brought to my attention some issues with this script. I have some changes locally that I will put into a new PR that you can merge in here.

Reply:
Is this PR ( https://github.com/rapidsai/cucim/pull/42 )?

Reply:
yes, @jakirkham. you beat me to it :slightly_smiling_face:. Ray and I had to fix some things in this script. We'll be pushing those changes to all the other repos as well.

Reply:
@ajschmidt8 cuCIM project depends on VERSION file that is used in building/packaging (cmake/python).
According to `update-version.sh`'s use (executed for each branching), I think getting the current tag (without 'v' or 'a' in it) from VERSION file does not impact on existing workflow. Please let me know if there is an issue with this.

Reply:
I think using the tag should be okay. I will test the `update-version.sh` script locally first before approving this PR.